code/modules/mob/living/carbon/human/say.dm

```diff
@@ -120,6 +120,31 @@
 /mob/living/carbon/human/proc/GetSpecialVoice()
 	return special_voice
 
+GLOBAL_LIST_INIT(soapy_words, list(
+	"shit" = "snap",
```

Comment:
I feel "snap" might not work for the intended purposes here---also not quite as comical as the others.

Reply:
Could always go for "shoot", for its phonetic similarity.